.github/workflows/codespell.yml

```diff
@@ -0,0 +1,23 @@
+# Codespell configuration is within .codespellrc
```

Comment:
This will not provide **GitHub Annotations** to contributors but using https://github.com/codespell-project/actions-codespell will.